client/my-sites/customer-home/main.jsx

```diff
@@ -49,6 +50,14 @@ const Home = ( {
 	trackViewSiteAction,
 } ) => {
 	const translate = useTranslate();
+	// use an effect to fire the exposure event only if the site id changes or first view
+	useEffect( () => {
+		if ( canUserUseCustomerHome ) {
+			recordTracksEvent( 'calypso_saw_a_a_test', {
```

Comment:
This needs to be connected in mapDispatchToProps, as I don't see this being fired.

To debug analytic events try setting in dev console: `localStorage.debug = 'calypso:analytics*'` and refresh the page.

We should see events like the following in console:
<img width="967" alt="Screen Shot 2020-05-12 at 2 39 05 PM" src="https://user-images.githubusercontent.com/1270189/81748250-5f9a3600-945e-11ea-9b2e-2bd73b2e4f7c.png">

Reply:
Good catch! I was wondering why I wasn't seeing that event but figured it was me or my environment.